change/office-ui-fabric-react-2019-11-25-16-27-13-teachingbubble-fix.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

This should be minor now that you're adding an API
```suggestion
  "type": "minor",
```